wordpress-mobile/WordPress-iOS
# Unified Prologue - Update intro screen using SwiftUI

Ref #15056 

This PR fixes layout issue in the intro screen of the Unified Prologue. A new SwiftUI implementation replaces the old UIKit version

To test:

- Build/run and logout if necessary
- Make sure the intro screen looks consistent with the designs (#15056). See also examples below. Test on a variety of conditions: iPhones and iPads, dark/light mode, portrait/landscape (iPad).


iPhone 12 Pro | iPad Pro 11 inch
|---|---|
<img height=400 src=https://user-images.githubusercontent.com/34376330/113385569-0412b880-934e-11eb-87fa-af66595093cd.png> | <img height=400 src=https://user-images.githubusercontent.com/34376330/113385587-083ed600-934e-11eb-9cd4-b7cbe13f1039.png>


## Regression Notes
1. Potential unintended areas of impact
This view is only used in the Intro screen of Unified Prologue

2. What I did to test those areas of impact (or what existing automated tests I relied on)
test it on a variety of devices to make sure it's consistent with the designs

3. What automated tests I added (or what prevented me from doing so)
NA

PR submission checklist:
